data/templates/slapd/mailserver.schema

```diff
@@ -82,7 +82,19 @@ objectclass ( 1.3.6.1.4.1.40328.1.1.2.2
 
 # Mail Group Objectclass
 objectclass ( 1.3.6.1.4.1.40328.1.1.2.3
-        NAME 'mailGroup' SUP top AUXILIARY
+        NAME 'mailGroup'
+        SUP top
+        AUXILIARY
         DESC 'Mail Group'
         MUST ( mail )
         )
+
+# Mail Alias Objectclass
+objectclass ( 1.3.6.1.4.1.40328.1.2.4.4
+        NAME 'mailAlias'
+        SUP top
+        STRUCTURAL
```

Comment:
- missing DESC
- after second thought, the existing but unused mailGroup was probably meant to be used for aliases. Look into deleting it ? or reusing it ?